src/pages/Account/Account.scss

```diff
@@ -120,6 +120,16 @@ $page-max-width: 1240px;
     height: 225px;
     background-size: 280px;
   }
+
+  &.github-plain {
+    margin: 0 auto;
+    background-image: url(/assets/images/Summit-logo-plain.png);
+    background-repeat: no-repeat;
+    background-position: center;
+    width: 580px;
+    height: 225px;
+    background-size: 400px;
+  }
```

Comment:
Does the old `/assets/images/Summit_Logo.png` still exist here? And does it need to? Is it used anywhere else? Or was it intended for this location/use case?

Reply:
Seems it's used in `src/pages/Account/Questions.tsx`

```
<div className="hero-logo github"></div>
```